tests/core/nogo/custom/return_bool_analyzer.go

```diff
@@ -0,0 +1,37 @@
+// returnboolanalyzer checks for functions that return bool.
```

Comment:
There's nothing wrong with returning a bool. It would be better to use an example that at least hinted at an actual problem.